docs/content/en/docs/fleet-manager/backup-plugin.md

```diff
@@ -0,0 +1,191 @@
+
+Set up the Fleet manager by following the instructions in the [installation guide](/docs/setup/install-fleet-manager/).
+
+### 2. Configuring Object Storage for Velero
+
+Kurator's backup relies on Velero. Hence, [Object Storage](https://velero.io/docs/v1.12/how-velero-works/) is essential to restore Kubernetes resources. We support various object storage options. In this guide, we provide detailed usage instructions for the following configurations:
+
+- [Minio](https://min.io/): Install a Minio service in the current Kurator host cluster for local validation. Refer to the [installation guide](/docs/setup/install-minio).
+
+- Cloud Service Provider's Object Storage (using Huawei Cloud OBS as an example): Purchase or use an existing object storage service.
+
+> **Note**: The Minio method is intended only for validation purposes. For production environments, it's recommended to use cloud service providers' storage services.
+
+After choosing the storage method, create a secret with the access details. Use this secret's name as a parameter for the fleet.
+
+For Minio: [Minio installation guide](/docs/setup/install-minio).
+
+For cloud storage using AK and SK (e.g., OBS):
+
+```console
+kubectl create secret generic obs-credentials --from-literal=access-key={YOUR_ACCESS_KEY} --from-literal=secret-key={YOUR_SECRET_KEY}
+```
+
+For more configuration options, please refer to the [Fleet API](https://kurator.dev/docs/references/fleet-api/).
+
+### 3. Secrets and Setup for Attached Clusters
+
+```console
+kubectl create secret generic kurator-member1 --from-file=kurator-member1.config=/root/.kube/kurator-member1.config
+kubectl create secret generic kurator-member2 --from-file=kurator-member2.config=/root/.kube/kurator-member2.config
+kubectl apply -f kurator/examples/backup/common/attachedcluster.yaml
+```
+
+## Create a Fleet with the Backup Plugin Enabled
+
+### Use Minio
+
+If you opt to use Minio, apply the following configuration example:
+
+```console
+kubectl apply -f examples/backup/fleet-minio.yaml
+```
+
+> **Note**: The `endpoint`(http://172.18.255.200:9000) in `fleet-minio.yaml` is depends on your minio service ip, check your minio service ip in [Minio installation guide](/docs/setup/install-minio).
+
+### Use OBS
+
+If you choose to use OBS, apply the following configuration example:
+
+```console
+kubectl apply -f examples/backup/fleet-obs.yaml
+```
+
+### Fleet Backup Plugin Configuration Explained
+
+Here's the details of `fleet-obs.yaml`:
+
+```yaml
+apiVersion: fleet.kurator.dev/v1alpha1
+kind: Fleet
+metadata:
+  name: quickstart
+  namespace: default
+spec:
+  clusters:
+    - name: kurator-member1
+      kind: AttachedCluster
+    - name: kurator-member2
+      kind: AttachedCluster
+  plugin:
+    backup:
+      storage:
+        location:
+          bucket: kurator-obs
+          provider: huaweicloud
+          endpoint: http://obs.cn-south-1.myhuaweicloud.com
+          region: cn-south-1
+        secretName: obs-credentials
```

Comment:
any guide on the secret format?

Reply:
already consider it, not sure if it is enough:

line55: For more configuration options, please refer to the [Fleet API](https://kurator.dev/docs/references/fleet-api/).

the content of Fleet API link:

![image](https://github.com/kurator-dev/kurator/assets/45359033/3cd70e0f-a452-4e10-b393-e11dc4ba4f64)